packages/aragon-api-react/README.md

```diff
@@ -1,6 +1,6 @@
-# aragonAPI for React
+# aragonJS - React Hooks
```

Comment:
```suggestion
# JavaScript - React API
```

I think using "Hooks" directly is a bit misleading, since this package is meant to be used by all React devs, not just those who want hooks.